fhir-search/src/main/java/com/ibm/fhir/search/context/impl/FHIRSearchContextImpl.java

```diff
@@ -132,6 +139,31 @@ public void setSummaryParameter(SummaryValueSet summary) {
     @Override
     public void setSearchResourceTypes(List<String> searchResourceTypes) {
         this.searchResourceTypes = searchResourceTypes;
+    }
 
+    /* (non-Javadoc)
+     * @see java.lang.Object#toString()
+     */
```

Comment:
Previously, we decided not to use these (and I try to clean them out as I find them)
```suggestion
```

Reply:
Updated with suggestion.